doc/en/how-to/assert.rst

```diff
@@ -112,7 +112,7 @@ If you want to check if a block of code is raising an exact exception type, you
             foo()
         assert excinfo.type is RuntimeError
 
-The :func:`pytest.raises` call will succeed, even though the function raises :class:`NotImplementedError`, because
+The :func:`pytest.raises` call will fail, even though the function raises :class:`NotImplementedError`, eventhough
```

Comment:
"Eventhough" is not a valid word in English

And it's incorrect to say the pytest.raises call succeeds